test/fixtures/track-long-names.json

```diff
@@ -40,6 +40,7 @@
       {
         "_eventName": "Testing in case someone sends us events ",
```

Comment:
can you help me understand how this test case does not invalidate the bug report?